ui/admin/app/routes/scopes/scope/targets/target/injected-application-credential-sources.js

```diff
@@ -0,0 +1,58 @@
+import Route from '@ember/routing/route';
+import { all } from 'rsvp';
+import { action } from '@ember/object';
+import { loading } from 'ember-loading';
+import { confirm } from 'core/decorators/confirm';
+import { notifySuccess, notifyError } from 'core/decorators/notify';
+
+export default class ScopesScopeTargetsTargetInjectedApplicationCredentialSourcesRoute extends Route {
+  // =methods
+
+  /**
+   * Loads all credential libraries under the current target.
+   * @return {Promise{[CredentialLibraryModel, CredentialModel]}}
+   */
+  beforeModel() {
+    const { injected_application_credential_source_ids: sourceIDFragments } =
+      this.modelFor('scopes.scope.targets.target');
+    return all(
+      sourceIDFragments.map(({ value }) => {
+        const isStatic = value.includes('cred');
+        if (isStatic) {
+          return this.store.findRecord('credential', value, {
+            reload: true,
+          });
+        } else {
+          return this.store.findRecord('credential-library', value, {
+            reload: true,
+          });
+        }
+      })
+    );
+  }
+
+  /**
+   * Returns the previously loaded target instance.
+   * @return {TargetModel}
+   */
+  model() {
+    return this.modelFor('scopes.scope.targets.target');
+  }
+
+  // =actions
+
+  /**
+   * Remove a credential library from the current target.
+   * @param {TargetModel} target
+   * @param {CredentialLibraryModel} credentialLibrary
```

Comment:
Can we update `credential library` here as well?

Reply:
ty for that catch! updated